Project.toml

```diff
@@ -7,58 +7,25 @@ AbstractPlotting = "537997a7-5e4e-5d89-9595-2241ea00577e"
 FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
```

Comment:
The version needs to be bumped, unfortunately I can't suggest it :)

https://github.com/JuliaPlots/Makie.jl/pull/455/files#diff-910a7b3ca0075d545dcec45cb7335ca9R3

Reply:
Which version?